client/src/app/navbar/navbar.component.html

```diff
@@ -1,6 +1,6 @@
 <nav class="navbar navbar-light bg-primary">
   <app-sidenav-toggler></app-sidenav-toggler>
-  <a class="navbar-brand mx-auto">Salud!</a>
+  <a routerLink="/" class="navbar-brand mx-auto">Salud!</a>
```

Comment:
routes can't start with '/'. I tried this too. the default route is ''. I set it to 'home' automatically. So change this to 'home'.